src/main/java/frc/robot/subsystems/ClimbPivot.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.CurrentLimitsConfigs;
+import com.ctre.phoenix6.configs.MotorOutputConfigs;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Hardware;
+
+public class ClimbPivot extends SubsystemBase {
+
+  private final TalonFX climbPivotMotorOne;
+  private final TalonFX climbPivotMotorTwo;
+
+  public ClimbPivot() {
+    climbPivotMotorOne = new TalonFX(Hardware.CLIMB_PIVOT_MOTOR_ONE_ID);
+    climbPivotMotorTwo = new TalonFX(Hardware.CLIMB_PIVOT_MOTOR_TWO_ID);
+
+    configureMotors();
+  }
+
+  private void configureMotors() {
```

Comment:
Also need to set one of the motors to follow the other, as I expect they'll be e mechanically linked